utils/hct/hctdb.py

```diff
@@ -5508,9 +5511,37 @@ def UFI(name, **mappings):
         )
         next_op_idx += 1
 
+        # RayQuery
+        self.add_dxil_op(
+            "AllocateRayQuery2",
+            next_op_idx,
+            "AllocateRayQuery2",
+            "allocates space for RayQuery and return handle",
+            "v",
+            "",
+            [
+                db_dxil_param(0, "i32", "", "handle to RayQuery state"),
+                db_dxil_param(
+                    2,
+                    "u32",
+                    "constRayFlags",
```

Comment:
Is this an existing convention to have a "const" prefix for parameter names like "constRayFlags" here?

Reply:
It is not an existing convention, this seems to be the only parameter with that prefix.

Reply:
The reason for constRayFlags was that additional non-const "RayFlags" are provided on the "TraceRayInline" which are combined (OR'd) with these flags, so this naming differentiates them.